.github/workflows/go.yml

```diff
@@ -97,6 +97,7 @@ jobs:
         - {name: "Timescaledb 1.x (pg 12)",  shortname: "tsdb1x-pg-12", ext: true,  tsdb: true,  tsdb2: false, tsdboss: false, multi: false, pg: 12}
         - {name: "Plain Postgres (12)",      shortname: "plain-pg-12",  ext: false, tsdb: false, tsdb2: false, tsdboss: false, multi: false, pg: 12}
         - {name: "TimescaleDB-OSS",          shortname: "tsdb-oss",     ext: false, tsdb: true,  tsdb2: true,  tsdboss: true,  multi: false, pg: 13}
+        - {name: "pg 14",                    shortname: "pg14",         ext: true,  tsdb: true,  tsdb2: true,  tsdboss: false, multi: false, pg: 14}
```

Comment:
Why are we only running one test config for pg14? Is that the only setup for 14 we will support?

Actually, in general, what is the thinking behind these? We have one config for 12, several for 13, and one for 14. How do we decide which combos to test?

Reply:
So the overall logic is we test most options on latest (OSS, plain postgres, etc) and then add some test for older versions with configs that make sense.  But we aren't quite ready to make pg14 "latest" b/c we are waiting on the timescaledb-single-ha release.